alot/settings/manager.py

```diff
@@ -77,13 +77,21 @@ def read_config(self, path):
         self._config.merge(newconfig)
         self._config.walk(self._expand_config_values)
 
-        hooks_path = os.path.expanduser(self._config.get('hooksfile'))
-        try:
-            spec = importlib.util.spec_from_file_location('hooks', hooks_path)
-            self.hooks = importlib.util.module_from_spec(spec)
-            spec.loader.exec_module(self.hooks)
-        except:
-            logging.exception('unable to load hooks file:%s', hooks_path)
+        # set up hooks module if requested
+        hooks_path = self._config.get('hooksfile')
+        if hooks_path:
+            hooks_path = os.path.expanduser(hooks_path)
+            logging.debug('hooks: loading from: %s', hooks_path)
+            try:
+                spec = importlib.util.spec_from_file_location('hooks',
```

Comment:
It seems that this function needs a path that ends in ".py" otherwise it will return `None`. So we can not advertise `hooksfile=/dev/null` to silence the logging.

We could instead advertise that the user should create an empty hooks file and put the path into the config. Not sure if that is to much noise for people who do not even use hooks.

Reply:
we could also have alot *create* an empty hooks file, or a stub, at the default location. But this would possibly a little overkill...

Reply:
We could also special case the empty string as an alternative to `None`. The default would be `None` and would result in a warning, the user can set the empty string to prevent the warning.  My filesystem does not accept the empty string as a file name so I assume this is "safe".

Reply:
All possible, yes.
Frankly, I would like to avoid introducing functionality to show warnings about this just to cater for current users and adding maintainance baggage for later down the line (to take that warning stuff out again once it is old).
This is what you are suggesting right?

Reply:
Yes that is what I am suggesting. 

My motivation is that I expect some breakage for users that is hard to understand or off putting (hooks not working silently). We could just announce this breaking change in the release notes and be done with it.